README.md

```diff
@@ -513,6 +537,15 @@ export default App = () => {
 };
 ```
 
+**Note:** If you use `basename` prop for `<BrowserRouter>`, use this implementation to fix `basename` duplication problem:
+```jsx
+  const restoreOriginalUri = async (_oktaAuth, originalUri) => {
+    const basepath = history.createHref({});
+    const originalUriWithoutBasepath = originalUri.replace(basepath, '/');
+    history.replace(toRelativeUrl(originalUriWithoutBasepath, window.location.origin));
```

Comment:
nit: add `import { toRelativeUrl } from '@okta/okta-auth-js';` in this code snippet.